src/main/java/org/zalando/nakadi/util/AuthorizationUtils.java

```diff
@@ -0,0 +1,79 @@
+package org.zalando.nakadi.util;
+
+import org.zalando.nakadi.domain.EventType;
+import org.zalando.nakadi.domain.EventTypeAuthorization;
+import org.zalando.nakadi.domain.SubscriptionBase;
+import org.zalando.nakadi.exceptions.InternalNakadiException;
+import org.zalando.nakadi.exceptions.runtime.AccessDeniedException;
+import org.zalando.nakadi.plugin.api.authz.AuthorizationAttribute;
+import org.zalando.nakadi.plugin.api.authz.AuthorizationService;
+import org.zalando.nakadi.plugin.api.authz.Resource;
+import org.zalando.nakadi.plugin.api.authz.Subject;
+import org.zalando.nakadi.repository.EventTypeRepository;
+import org.zalando.nakadi.security.Client;
+
+import java.util.List;
+import java.util.Optional;
+
+public class AuthorizationUtils {
+    public static Resource createStreamResource(final EventType eventType) {
+        return new Resource() {
+            @Override
+            public String getName() {
+                return eventType.getName();
+            }
+
+            @Override
+            public String getType() {
+                return "event-type";
+            }
+
+            @Override
+            public Optional<List<AuthorizationAttribute>> getAttributesForOperation(
+                    final AuthorizationService.Operation operation) {
```

Comment:
yeah. I'll maybe just use the resource you created in your PR. I'll wait for it to be merged and then fix it here, what do you think?